botocore/config.py

```diff
@@ -21,6 +22,8 @@
     InvalidS3AddressingStyleError,
 )
 
+logger = logging.getLogger(__name__)
```

Comment:
Don't see it used below, maybe leftover from debugging?

Reply:
Fixed in e9c47aba9350cbc77e2cbcdee1cc2b1763db2a0d